src/chapters/setup-intro-data-types/why-java.rst

```diff
@@ -1,2 +1,21 @@
-Why Learn Another Language?
-============================
+Why Learn Java?
+===============
+
+This course is an introduction to Java and the Spring Boot framework. Java is a
+powerful programming language and one of the most widely used across the globe. 
+This course is designed for learners who are already familiar with at least one
+programming language. Thus, we'll move quickly through the syntax rules and
+procedural basics of the language (ie, how does one write a for loop in Java? a 
+conditional statement?). 
+
+As you've likely heard before, once you grasp the fundamentals of programming in 
+one language, learning another becomes much easier. Professional software 
+developers often work in environments involving several programming languages so 
+it's wise to start learning new ones early on in your career. 
+
+Beyond the basics of programming in Java, you'll learn some key **object oriented**
```

Comment:
Our style guide reserves **bold** for vocab words.  If you do not provide a definition for the terms, you can use *italics* for emphasis.